scripts/00_training/computer_vision.py

```diff
@@ -0,0 +1,300 @@
+# %%
+from accelerate import Accelerator
+from accelerate.utils import set_seed
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from src.model import MLP, LeNet5
+
+from torchvision import datasets, transforms
+import torchvision.models as models
+
+import os
+import json
+import wandb
+import evaluate
+from tqdm import trange
+import argparse
+
+import numpy as np
+
+
+from src.metrics import get_metrics_mlp, get_metrics_lenet5, get_metrics_resnet18
+from src.model import MyResNet, MyBasicBlock
+
+# %%
+# replicating https://arxiv.org/pdf/2210.01117.pdf
+
+
+# %%
+def get_dataloaders(dataset_name, model_name, train_bsz, test_bsz, train_subsample):
+    if dataset_name == 'mnist':
+        if model_name == 'mlp':
+            transform = transforms.Compose([transforms.ToTensor(), transforms.Lambda(lambda x: torch.flatten(x))])
+        else:
+            transform = transforms.Compose([
+                transforms.Resize((32,32)),
+                transforms.ToTensor(),
+                transforms.Normalize(mean = (0.1307,), std = (0.3081,)),
+            ])
+        train_data = datasets.MNIST(
+            "./data", train=True, download=True, transform=transform
+        )
```

Comment:
This `"./data"` directory isn't included in this repo, nor is it mentioned in the README. Can you (1) provide instructions in the README as to where to find/download the data, and (2) change this to a command-line argument (with a default value) so that it isn't hard-coded?